### ISSUE TITLE
[Usable – Make Code – Share] Narrator announces 'Project name' information twice while navigating in scan mode under 'Share project' pop-up.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3594]
**User Experience:**
Screen reader user will be confused, if narrator is reading same information two times and also it will take time to navigate another control over the screen.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Share" option appearing in header section and select it.
6. Navigate through all the controls of  "Share Project" dialog.
5. Verify that Narrator announces 'Project name' information twice while navigating in scan mode or not.

**Actual Result:**
Narrator announces 'Project name' information twice while navigating in scan mode under 'Share project' pop-up.

**Expected Result:**
Narrator should announce 'Project name' information only once while navigating in scan mode under 'Share project' pop-up.

**Note:**
1. This issue does not exists on Firefox & Chrome.
2. Same issue exists on 'Name' edit field under 'Project Setting' screen.

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0)
Screen Reader: Narrator

**Comment by Shalini3457** [comment]
Labels:
#A11yUsable, #Accessibility, #A11ySev4, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #Win10-Edge(Chromium)

[Usable_Share project_Twice information.zip](https://github.com/microsoft/pxt-microbit/files/5493443/Usable_Share.project_Twice.information.zip)

**Comment by jwunderl** [comment]
Only reads out once now, likely fixed in edge update
